bin/main

```diff
@@ -1,22 +1,96 @@
 #!/usr/bin/env ruby
-# frozen_string_literal: true
 
+# Class to make users
 class User
   attr_reader :name
 
   attr_writer :marker
+
   def initialize(name)
     @name = name
-    @marker = ''
   end
 
-  def marker(marker)
-    @marker = marker
+  def mark
+    @marker
   end
 end
-puts 'Enter name'
-player1 = User.new(gets.chomp)
-puts 'Choose marker'
-player1.marker(gets.chomp)
+turn = true
+
+puts "Welcome to Tic-Tac-Toe\n\n"
+puts ' X | X |   '
+puts '---+---+---'
+puts ' O | O | O '
+puts '---+---+---'
+puts '   | O | X '
+answer = ''
+cells = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+until answer == 'yes'
```

Comment:
You can ask users to put something simple like `Y or N` instead of `Yes or No` remember that users are lazy people

Reply:
Changed "yes" to "y"